src/eko/io/types.py

```diff
@@ -63,6 +65,14 @@ class CouplingsRef(DictLike):
 
     alphas: FloatRef
     alphaem: FloatRef
+    num_flavs_ref: Optional[FlavorsNumber]
+    r"""Number of active flavors at strong coupling reference scale.
+
+    I.e. :math:`n_{f,\text{ref}}(\mu^2_{\text{ref}})`, formerly called
+    ``nfref``.
+
+    """
+    num_flavs_max_as: FlavorsNumber
```

Comment:
```suggestion
    max_num_flavs: FlavorsNumber
```
`as` has become redundant and word order would be more English

Reply:
Done in e888119543e20d3d574444426729426ac27ba959